packages/block-editor/src/index.js

```diff
@@ -12,6 +12,7 @@ export {
 	getSpacingClassesAndStyles as __experimentalGetSpacingClassesAndStyles,
 	getGapCSSValue as __experimentalGetGapCSSValue,
 	getShadowClassesAndStyles as __experimentalGetShadowClassesAndStyles,
+	getDimensionsClassesAndStyles as __experimentalGetDimensionsClassesAndStyles,
```

Comment:
I'm wondering whether we should use the `__experimental` prefix.

Reply:
Good question. 

The only reason I see to use experimental here is to advertise to third parties that the public method is subject to change, and should not be relied upon long term.

Is that the case here?

The counter argument is that anything can and will change: should we add `__experimental*` to everything? 😄 

For my part, I don't see an obvious case for the experimental flag — it's possible folks have followed the naming pattern of other exported hooks in this file without considering it. 

If third parties shouldn't be using this API then it could also be exported in the private APIs.

Reply:
Agreed 👍 

I believe the other functions (maybe shadow excluded) were created before the private APIs approach solidified. 

As part of other separate block support stabilization efforts, these utils are also on my radar to stabilize.